src/methods/createAndSendTx.ts

```diff
@@ -1,9 +1,9 @@
 import { ApiPromise, WsProvider } from "@polkadot/api";
-import { isHex, u8aToHex } from "@polkadot/util";
+import { u8aToHex } from "@polkadot/util";
 import { typesBundle } from "moonbeam-types-bundle";
 import { ISubmittableResult, SignerPayloadJSON } from "@polkadot/types/types";
 import prompts from "prompts";
-import { exit, moonbeamChains } from "./utils";
+import { moonbeamChains } from "./utils";
 import { SignerResult, SubmittableExtrinsic } from "@polkadot/api/types";
 
 export async function createAndSendTx(
```

Comment:
It should be split into 2 objects for better readability (instead of having 7 arguments most of them hard to guess from the title of the function):
- one about the tx and the data you want to send
- one related to the network